website/docs/wallet/withdraw-validator.md

```diff
@@ -190,8 +190,14 @@ By calling the command above, you should go through an interactive process that
 1. **Your mnemonic language**. You can see the different options available, where English is one of the options, among others
 2. **The network** you wish to perform this operation for. Select the `zhejiang` network for this tutorial
 3. Enter your **mnemonic** next
-4. Next, you will be asked for your the index of the withdrawal key you wish to use. For **most users**, this will be 0 for the first validator, 1 for their second, etc....
-5. You will then be asked the **validator** **indices** for the validators you wish to generate the message for. You can find your validator indices on block explorers such as [https://beaconcha.in](https://beaconcha.in) or in your Prysm validator client logs. For example, the validator with public key `0x8078c7f4ab6f9eaaf59332b745be8834434af4ab3c741899abcff93563544d2e5a89acf2bec1eda2535610f253f73ee6` on [https://beacocha.in](https://beacocha.in) has validator index 8 by navigating to its [page](https://beaconcha.in/validator/8)
+4. Next, you will be asked for your starting index when you created your validators,read more about hd wallets [here](https://eips.ethereum.org/EIPS/eip-2334#path). For **most users**, this will be 0 unless you created validators from a non default index.
+
+:::info
+The validator starting index can be found by looking into the original `deposit.json` file used for staking. Inside the json file you can count each validator's public key in sequential order starting from 0.
+Based on which validator you would like to withdraw in step 5. The starting index may differ if you are skipping some validators to withdraw from. There are other niche cases where the mnemonic is used for deposit generation multiple times resulting in a different validator starting index.
```

Comment:
This sentence is a bit iffy "Based on ...", I tried to fix it a bit but it still seems a bit off to me, maybe you have a better idea on how to improve it:
"The starting index may differ if you are not withdrawing all of your validators in step 5 but are instead skipping some. There are also other niche cases where the mnemonic is used multiple times for deposit generation resulting in a different starting index."

Reply:
Also should we add something like "Therefore please make sure to double check your starting index." or is this mainly just as an FYI.

Reply:
this is just a FYI